job_executor/model/datastore_versions.py

```diff
@@ -50,7 +50,9 @@ def add_new_release_version(
         new_release_version = DatastoreVersion(
             version=new_version_number,
             description=description,
-            release_time=(datetime.now() - datetime.utcfromtimestamp(0)).days,
+            release_time=(
+                (datetime.now() - datetime.utcfromtimestamp(0)).seconds
```

Comment:
`seconds` here whereas `total_seconds()` above, nice to use one way

Reply:
oops missed one! Proves your point about a function tho 👍🏻 I'm on it